GridTools-Glossary.md

```diff
@@ -16,9 +18,17 @@ Describes a set of [grid points](#grid-point) on which the computation is applie
 
 ### Iteration Space
 
+TODO disambiguate from compute domain or make it an alias.
+
 ### Origin
 
-### Unstructured
+### Structured Grid
+
+A grid where cells follow a regular pattern that allows strided access to neighbors. Examples of structured grids are [Cartesian grids](#Cartesian-Grid).
```

Comment:
Can you add other structured non-cartesian grid examples (maybe icosahedral grids, I don't know)?

Reply:
See my additions.